docs/pages/menu.md

```diff
@@ -142,6 +136,39 @@ Add the `.vertical` class to a Menu to switch its orientation.
 </ul>
 ```
 
+<br>
+
+Add `align-right` class for making the vertical menu aligned to the right. 
+
+```html_example
+<ul class="vertical menu align-right">
+  <li><a href="#">One</a></li>
+  <li><a href="#">Two</a></li>
+  <li><a href="#">Three</a></li>
+  <li><a href="#">Four</a></li>
+</ul>
+```
+
+<br>
+
+Add `align-center` class for making the vertical menu aligned to the center.
+
+<div class="warning callout">
+  <p>
+    The above vertically left &amp; right aligned menu are supported in all types of menu's. <br>
+    But as we don't think `align-center` is useful for dropdown, accordion or a drilldown menu, we have just kept it on basic menu only.
```

Comment:
I think stating what we think is useful is not necessary... lets just state that `align-center` just works in basic menus.